deluge/log.py

```diff
@@ -107,7 +107,7 @@ def findCaller(self):  # NOQA
 }
 
 
-def setup_logger(level="error", filename=None, filemode="w", logrotate=None):
+def setup_logger(level="error", filename=None, filemode="w", logrotate=None, twisted_observer=True):
```

Comment:
Added comment on commit message:
"Added parameter log.setup_logger to prevent output noise in unit tests"

Is that good?